frontend/src/components/grouping/FuzzyMatchNotificationGroupingComponent.vue

```diff
@@ -208,27 +215,21 @@ fuzzyBus.on(handleFuzzyErrorMessage);
     :title="fuzzyMatchedError.fuzzy ? 'Possible matching records found' : 'Client already exists'"
   >
     <div>
-      <span class="body-compact-02">
-        <template v-if="fuzzyMatchedError.fuzzy">
-          {{ uniqueClientNumbers.length }} similar client
-          <span v-if="uniqueClientNumbers.length === 1">record was</span>
-          <span v-else>records were</span>
-          found. Review their information in the Client Management System to determine if you should
-          should create a new client:
-        </template>
-        <template v-else>
-          Looks like ”{{ businessName }}” has a client number. Review their information in the
-          Management System if necessary:
-        </template>
-      </span>
-      <ul>
-        <!-- eslint-disable-next-line vue/no-v-html -->
-        <li
-          v-for="match in fuzzyMatchedError.matches"
-          :key="match.result.field"
-          v-dompurify-html="getListItemContent(match)"
-        ></li>
-      </ul>
+      <!-- eslint-disable-next-line vue/no-v-html -->
+      <span 
+        class="body-compact-01"
+        v-if="fuzzyMatchedError.fuzzy"
+        v-for="match in fuzzyMatchedError.matches"
+        :key="match.result.field"
+        v-dompurify-html="getListItemContent(match)"
+      ></span>
+        <span class="body-compact-01" v-if="fuzzyMatchedError.fuzzy">
+          Review them in the Client Management System to determine if you should create a new client.
+        </span>
+        <span v-else class="body-compact-01">
+          It looks like ”{{ businessName }}” has a client number 
```

Comment:
There's an extra word ("a") in "has **a** client number".
It should be "has client number".

Reply:
My bad, was due to habit 😛